src/metrics.js

```diff
@@ -68,15 +49,48 @@ function sendFeatureMetric(name) {
 function featureRequested(name) {
   sendFeatureMetric(`atom_${name}_requested`);
 }
+
 function featureFulfilled(name) {
   sendFeatureMetric(`atom_${name}_fulfilled`);
 }
 
+function track(event, props = {}) {
+  const e = {
+    event,
+    userId: '0',
+    user_id: distinctID(),
+    sent_at: Math.floor(new Date().getTime() / 1000),
+    source: 'vscode',
+  };
+
+  for (const k in props) { e[k] = props[k]; }
+
+  Logger.debug('segment:', e);
+
+  if (process.env.NODE_ENV !== 'test') { ANALYTICS.track(e); }
```

Comment:
@abe33 I checked events at https://app.segment.com/kite-juan/sources/vscode_development/debugger.  As far as I can understand the custom properties should be send in "properties", they are currently send as top-level properties.

I think `props` should be send as 2nd argument, as seen on https://segment.com/docs/sources/website/analytics.js/#track

Current JSON on segment.com (with `props` on top-level):
```json
{
  "_metadata": {
    "nodeVersion": "7.9.0"
  },
  "context": {
    "library": {
      "name": "analytics-node",
      "version": "3.1.1"
    }
  },
  "event": "kited_health",
  "messageId": "node-MEBo3wUNtyGC8AVRfQ0goRAgUhH6gFID",
  "os_name": "windows",
  "plugin_version": "0.0.1",
  "sent_at": 1511365715,
  "source": "vscode",
  "timestamp": "2017-11-22T15:48:36.706Z",
  "type": "track",
  "userId": "0",
  "user_id": "7d08652df01f7d0d0758c574132e2db80dbba94b04ea2093275236c7b641f097",
  "value": "activated",
  "writeKey": "tlsFlkyXKAyTtbIYMsx8slXJxDQv8Izn",
  "sentAt": "2017-11-22T15:48:35.598Z",
  "integrations": {},
  "receivedAt": "2017-11-22T15:48:36.706Z",
  "originalTimestamp": "2017-11-22T15:48:35.598Z"
}
```